### DISCUSSION TITLE
Add required API permissions to documentation

### DISCUSSION THREAD

**Discussion by electricgull** [community/community#69917] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

There have been a few instances where I have wasted time debugging or working with support on what I thought was a bug in an API only to find out it was some obscure required permission. For example, I opened this discussion question (and support ticket) 
https://github.com/orgs/community/discussions/69841

I was making a GET request to an API. I had set the read:org permissions, assuming that should be enough for a GET request. I was wrong and wasted a whole trying to figure out the issue. Come to find out (thank you GitHub support) the problem was easily solved by providing admin:org permissions. If GitHub support has access to this information, it seems like it would be extremely easy to add this information to the API documentation / Swagger file so it can be referenced at https://docs.github.com/en/rest.

**Comment by ghostinhershell** [comment]
Hey @electricgull! 

The information that you've learned along the way sounds like it could be helpful for the whole GitHub community. If it's not already covered in our docs (it sounds like it isn't), please open an [issue](https://github.com/github/docs/issues/new/choose) and provide any helpful details for the review process. Our docs are open sourced! Everyone in our community is welcome to [contribute](https://github.com/github/docs/blob/main/CONTRIBUTING.md#welcome-to-github-docs-contributing-guide-) and help users all over the world.

Please let us know if you have any additional questions. Cheers!
